base.php

```diff
@@ -28,6 +28,7 @@
 $has_sticky_template = apply_filters( 'crio_premium_get_sticky_page_header', get_the_ID() );
 $has_sticky_template = get_the_ID() === $has_sticky_template ? false : $has_sticky_template;
 
+if ( ! isset( $_REQUEST[ 'pdf' ] ) && ! isset( $_REQUEST[ 'genpdf'] ) ) {
```

Comment:
Isn't this going to break any url that has `pdf=` or `genpdf=` parameters?

You're also stating that any site using Crio can't have a form submitted via post with a `pdf` value? What if someone is uploading a pdf to a site, and the form is using `<input type="file" name="pdf" id="pdf" ... />`

Reply:
The logic for this has been changed, and moved out of the base.php file.